sdk/src/Vyper.ts

```diff
@@ -8,17 +8,39 @@ import { LastUpdate } from "./LastUpdate";
 import { ReserveFairValue } from "./ReserveFairValue";
 import { TrancheData } from "./TrancheData";
 import { TrancheFairValue } from "./TrancheFairValue";
+import { RedeemLogicLending } from "../../target/types/redeem_logic_lending";
```

Comment:
the plugin typescript class and the respective idl json file needs to be imported in a new plugin wrapper that we can use to call istructions to the plugin. please beaware that the plugins can have multiple instruction (you can check current plugins ixs from the programs folder) so we'll need to be able to call those ixs from a typescript wrapper class